media/CircleGraph/external/base.js

```diff
@@ -1,3 +1,5 @@
+// This file is copied from
+// https://github.com/lutzroeder/netron/blob/ae449ff55642636e6a1eef092eda34ffcba1c684/source/base.js
```

Comment:
Adding something like this is strict requirement from our open source division?
When we have to revise to next version then we have to add all these comments for these and that I think is a waste, which I don't like to do.

Reply:
> Adding something like this is strict requirement from our open source division?

Unfortunately, yes it is IIUC.
Actually, I wrote about this as "Add copyright & license statement of netron, origin at the top of the file." in https://github.com/Samsung/ONE-vscode/pull/518#issuecomment-1111790784 :(

Reply:
Could you invide the person in charge for this case. Personlly I would like to know his option as you wrote `IIUC` not `I know`

CC @lemmaa

Reply:
I left comment about this at internal secret repo issue(168)